plenum/test/replica/test_process_prepare.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+
+from plenum.common.exceptions import SuspiciousNode
+from plenum.common.util import updateNamedTuple
+from plenum.server.suspicion_codes import Suspicions
+from plenum.test.helper import generate_state_root
+
+nodeCount = 7
+
+NON_PRIMARY_NAME = "Gamma:0"
+
+
+@pytest.fixture(scope='function', params=[0])
+def viewNo(tconf, request):
+    return request.param
+
+
+@pytest.fixture(scope="function",
+                params=['BLS_not_None'])
+def multi_sig(fake_multi_sig, request):
+    if request.param == 'BLS_None':
+        return None
+    return fake_multi_sig
+
+
+@pytest.fixture(scope='function', params=['Primary', 'Non-Primary'])
+def r(replica_with_requests, request):
+    if request == 'Primary':
+        replica_with_requests.name = replica_with_requests.primaryName
+    else:
+        replica_with_requests.name = "Beta:0"
```

Comment:
I think it would make tests more clear if:
- "Alpha:0" was added to constants as `PRIMARY_NAME` (and tested in fixture to make sure that it's really primary)
- "Beta:0" was added tp constants as `OTHER_NON_PRIMARY_NAME`, so that all "actor" real names are in one place

Reply:
Agree